LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020 Weyert de Boer
```

Comment:
I dont think this is allowed, you'd need to keep the original one
https://www.tldrlegal.com/license/mit-license